### PR TITLE
Stop generating LanguageServiceRegistrationFiles

### PR DESCRIPTION
**Pull request by tmat**
These were only needed for classic setup.

### CODE REVIEW

**Comment by tmat** [comment]
@jasonmalinowski @jmarolf @basoundr

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb [hunk 1]
```diff
@@ -792,37 +792,20 @@ Public Class BuildDevDivInsertionFiles
...
-                                Case ".vsixmanifest"
-                                    RewriteVsixManifest(absoluteOutputFilePath)
-                            End Select
-                        Else
-                            ' paths are relative to input directory:
-                            filesToInsert.Add(New NugetFileInfo(partFileName))
-                            AddXmlDocumentationFile(filesToInsert, partFileName)
-                        End If
+                        ' paths are relative to input directory:
+                        filesToInsert.Add(New NugetFileInfo(partFileName))
```
Comment (jasonmalinowski) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: You've verified this thing still needs to do...whatever it does?

Comment (tmat) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: Yeah. It produces everything as before except for LanguageServiceRegistrationFiles

**Comment by basoundr** [comment]
:shipit:

File: src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb (hunk 1 above)
Comment (jasonmalinowski) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: @tmat: I guess I phrased my question poorly. Whatever it's doing, does it still need to be done? Wondering if there's more to delete. :-)

Comment (tmat) [inline src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb]: I don't think we can delete more at this point.

**Pull request merged**